allennlp/tests/fixtures/basic_classifier/common.jsonnet

```diff
@@ -19,7 +19,8 @@
     "iterator": {
         "type": "bucket",
         "sorting_keys": [["tokens", "num_tokens"]],
-        "batch_size": 5
+        "batch_size": 5,
+        "padding_noise": 0.0
```

Comment:
Is `"padding_noise": 0.0` necessary now? I'd like to avoid breaking existing config if at all possible.

Reply:
These were just spurious, non-deterministic test failures which I fixed (they were failing in model test where we compare 2 datasets read from file are the same), so I removed the non-determinism in the bucketing process here. Same below.

Reply:
Sorry if I'm being dense, but why did this only start now? I don't see spurious failures on http://build.allennlp.org/viewType.html?buildTypeId=AllenNLP_AllenNLPPullRequests&branch_AllenNLP=%3Cdefault%3E&tab=buildTypeStatusDiv.

Reply:
The build began failing at  https://github.com/allenai/allennlp/pull/3386/commits/28335cde7f2179eaaa2b2fbc99a757608f989da9:

http://build.allennlp.org/viewLog.html?buildId=21427&buildTypeId=AllenNLP_AllenNLPPullRequests&tab=buildLog&_focus=1481

3 tests failed: all of them were because differences not in the model but checking that the iterators load an identical first batch, which happens here:

https://github.com/allenai/allennlp/blob/master/allennlp/common/testing/model_test_case.py#L122


The batch elements are identical but in a different order (I checked). This test was essentially equivalent to checking the order of a list which with 0.1 prob has it's order changed.

The next commit https://github.com/allenai/allennlp/pull/3386/commits/285d60a13beffa70a7fac82ee96341f4a44c6ad3, also failed:

http://build.allennlp.org/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=21428&_focus=1635

This time it was the NaQaNet test, which is already flaky - the reason this test fails so much is because it has a lot of layers with relus in it, see here: 
https://github.com/allenai/allennlp/blob/master/allennlp/modules/seq2seq_encoders/qanet_encoder.py#L202